go.mod

```diff
@@ -3,21 +3,30 @@ module github.com/vaikas/ftp
 go 1.15
 
 replace (
-	k8s.io/api => k8s.io/api v0.18.8
-	k8s.io/apimachinery => k8s.io/apimachinery v0.18.8
-	k8s.io/client-go => k8s.io/client-go v0.18.8
+	k8s.io/api => k8s.io/api v0.18.12
```

Comment:
Can we get rid of these replaces? I'd prefer that we use 19.7 all over.
Just like here:
https://github.com/knative-sandbox/eventing-rabbitmq/pull/213

Reply:
sure